tests/common/include/tests/common/tools/detector_metadata.hpp

```diff
@@ -41,9 +41,59 @@ using cylinder = mask<cylinder2D<>, volume_link_type>;
 using disc = mask<ring2D<>, volume_link_type>;
 using unbounded_plane = mask<unmasked, volume_link_type>;
 
+/// material types
 using slab = material_slab<detray::scalar>;
 using rod = material_rod<detray::scalar>;
 
+/// surface grid types (regular, open binning)
+/// @{
+
+// surface grid definition: bin-content: std::array<dindex, 9>
+template <typename grid_shape_t, typename container_t>
+using surface_grid_t = grid<coordinate_axes<grid_shape_t, false, container_t>,
+                            dindex, simple_serializer, regular_attacher<9>>;
+
+// cylindrical grid for the barrel layers
+template <typename container_t>
+using cylinder_sf_grid = surface_grid_t<cylinder2D<>::axes<>, container_t>;
+
+// disc grid for the endcap layers
+template <typename container_t>
+using disc_sf_grid = surface_grid_t<ring2D<>::axes<>, container_t>;
+
+/// @}
+
+/// Material grid types
+/// @{
+
+// material grid definition: bin-content: material slab
+template <typename grid_shape_t, typename container_t>
```

Comment:
Why are the materials in grid even thought there are surface finders?

Reply:
Ah, sorry I wanted to remove this again. This was just to see if the grids actually also compile for all surface shapes, for material maps